Alexey_Michaylov/public_html/components/Linklist/Linklist.jsx

```diff
@@ -0,0 +1,27 @@
+import './Linklist.css';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default class Linklist extends Component {            
+    static propTypes = {
+        items: PropTypes.array.isRequired,
+        column: PropTypes.oneOf(['2','3']),
+        addclass: PropTypes.string,
+    }
+    
+    render() {
+        const { items, column, addclass } = this.props;
+        const listClasses = classNames({
+            'list-unstyled': true,
+            'two-column': column === '2',
+            'three-column': column === '3',
+        }, addclass);
```

Comment:
```suggestion
        }, className);
```
Рекомендую использовать стандартное имя проперти для этого) Более интуитивно получается